### PR TITLE
One more pass over clarifying licensing implications and license tracking in RAT

### PR DESCRIPTION
**Pull request by rvs**
Please take a look

### CODE REVIEW

**Comment by linwen** [comment]
Looks great! Thank you, Roman.

**Comment by changleicn** [comment]
Look good except that it does not have a JIRA number related to this.

**Comment by rvs** [comment]
@changleicn ah! Good point about the JIRA. Sorry about not filing one -- will keep that in mind.

File: src/include/nodes/plannerconfig.h [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (wangzw) [inline src/include/nodes/plannerconfig.h]: build error, should use C comments

File: src/include/nodes/stack.h [hunk 2]
```diff
@@ -1,3 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (wangzw) [inline src/include/nodes/stack.h]: build error, should use C comments

File: src/include/catalog/gp_version.in [hunk 3]
```diff
@@ -1,3 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (wangzw) [inline src/include/catalog/gp_version.in]: build error, should use C comments

**Comment by rvs** [comment]
Darn! Looks like my header injection script got tripped up for some reason :-(

**Pull request closed without merging**